fontbe/src/os2.rs

```diff
@@ -10,6 +12,187 @@ use crate::{
     orchestration::{BeWork, Context},
 };
 
+/// Used to build a bitfield.
+///
+/// Intended use: use the outer range to determine if a given bit might apply. If it might,
+/// check the inner ranges.
+///
+/// Generated by resources/scripts/os2_unicode_ranges.py.
+///
+/// <https://learn.microsoft.com/en-us/typography/opentype/spec/os2#ur>
+/// <https://github.com/fonttools/fonttools/blob/47813b217c1f8bc343c094776020b4f32fc024b0/Lib/fontTools/ttLib/tables/O_S_2f_2.py#L351-L557>
+const UNICODE_RANGES: &[(usize, usize, usize)] = &[
```

Comment:
is `usize` the best type to represent a codepoint? `char` might be semantically more accurate, and `u32` would be a more obvious choice if we want to use an integer type.

Also I wonder if it makes sense to represent the ranges as an actual range? maybe not, but just a thought.

Reply:
Fixed